tasks/libs/ciproviders/github_api.py

```diff
@@ -378,6 +394,29 @@ def create_release(self, tag, message, draft=True):
             draft=draft,
         )
 
+    def get_codereview_complexity(self, pr_id: int) -> str:
+        """
+        Get the complexity of the code review for a given PR, taking into account the number of files, lines and comments.
+        """
+        pr = self._repository.get_pull(pr_id)
+        criteria = {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Can you add a link to the documentation/details?